src/components/players/list/index.js

```diff
@@ -43,38 +45,26 @@ const List = ({
       const isSelected = person_id && selectedPlayerId === person_id;
 
       return (
-        <div
+        <Card
           key={index}
-          onClick={() => {
-            history.push(`/${urlName}/players/${person_id}`);
-            getSelectedPlayer(player);
-          }}
-          className={classnames(styles.playerCard, isSelected && styles.selectedCard)}>
-          <div className={styles.imageContainer}>
-            <img
-              src={person_id ? formatPlayerPhotoUrl(teamId, person_id) : placeholderImg}
-              alt="player headshot"
-              onError={(e) => (e.target.src = placeholderImg)}
-            />
-          </div>
-          <div style={{ borderColor: teamColor }} className={styles.imageLine} />
-          <div style={{ backgroundColor: isSelected && teamColor }} className={styles.detailsContainer}>
-            {!!Object.keys(player).length && (
-              <>
-                <div className={styles.number}>{jersey_number}</div>
-                <div className={styles.details}>
-                  <div className={styles.name}>
-                    {first_name} {last_name}
-                  </div>
-                  <div className={styles.position}>{position_full.replace('-', ' - ')}</div>
-                  <div className={styles.size}>
-                    {height_ft}' {height_in}, {weight_lbs} lbs
-                  </div>
-                </div>
-              </>
-            )}
-          </div>
-        </div>
+          personId={person_id}
+          firstName={first_name}
+          lastName={last_name}
+          jerseyNumber={jersey_number}
+          position={position_full}
+          heightInFeet={height_ft}
+          heightInInches={height_in}
+          weight={weight_lbs}
+          urlName={urlName}
+          teamId={teamId}
+          teamColor={teamColor}
+          isSelected={isSelected}
+          getSelectedPlayer={getSelectedPlayer}
+          player={player}
+          history={history}
+          formatPlayerPhotoUrl={formatPlayerPhotoUrl}
+          placeholderImg={placeholderImg}
+        />
```

Comment:
There are too many props being passed into Card component. This creates a lot of extra extra code. It should look more like this:

```
  return (
    <div ref={domRef} className={classnames(styles.container, isLoading && styles.noScroll)}>
      <Overlay isLoading={isLoading}>
        <Spinner height={19} width={4} radius={3} isLoading={isLoading} />
      </Overlay>
      {players.map((player, index) => (
        <Card
          key={index}
          getSelectedPlayer={getSelectedPlayer}
          selectedTeam={selectedTeam}
          selectedPlayerId={selectedPlayerId}
          player={player}
        />
      ))}
    </div>
  );
```

Complete the following:

- [ ] destructuring should happen in Card component
- [ ] `formatPlayerPhotoUrl`, `placeholderImg` are imported in Card component instead of passed as a prop
- [ ] `withrouter` should be removed from List component and added directly in Card component
- [ ] Card component will now calculate the `isSelected` boolean, instead of List Component
- [ ] There is no need for a `const renderPlayers`, instead lets move this `player.map` code directly into jsx